ms-utils/math/linear_algebra/cuda/CudaSparseSolve.hh

```diff
@@ -52,6 +52,9 @@ class CudaSparseSolve : public CholeskyLdu, public CudaIfUtils
         virtual ~CudaSparseSolve();
         /// @brief Changes matrix [A] into sparse form on the GPU but does not decompose it.
         virtual void Decompose(double* A, int n);
+        /// @brief Decomposes only the rows of the admittance matrix [A] given by the vector
+        ///        argument r.
+        void Decompose(double *A, int n, std::vector<int>& r) override { CholeskyLdu::Decompose(A, n, r); }
```

Comment:
Probably need unit test for this new function, also if possible move the implementation to an inline down below.

Reply:
Same comment as in CudaDenseDecomp. Also, note to self: forgot to remove _override_ here.

Reply:
Added unit test, copied from CholeskyLdu.